content/applications/finance/fiscal_localizations/belgium.rst

```diff
@@ -0,0 +1,164 @@
+=======
+Belgium
+=======
+
+.. _belgium/introduction:
+
+Fiscal certification: POS restaurant
+====================================
+
+In Belgium, the owner of a business that offers cooking services (such as a restaurant, a
+food-truck, etc.) and has a turnover (excluding VAT, drinks and take-away food) that exceeds 25 000
+euros a year, is required, by law, to issue their cash register receipts using a
+government-certified **Cash Register System**.
+
+This government-certified system entails the use of a **certified POS system**, along with a device
+called a **Fiscal Data Module** (or **black box**) and a **VAT Signing Card** (delivered after
+registration by the Service Public Federal Finances).
+
+A certified POS system
+----------------------
+
+To ensure full compliance with the law, the Odoo POS system is certified for `Odoo 9 (Odoo Online)
+<https://www.systemedecaisseenregistreuse.be/systemes-certifies>`_, and Odoo 13 to Odoo 15 (**Odoo
+Online** and **odoo.sh**).
+
+A certified POS system must adhere to rigorous government regulations, which means it operates
+differently from a non-certified POS.
+
+- You cannot:
+
+  - Make a refund.
```

Comment:
Since 15, it's possible to make a refund.
(It was not possible when it was done from the back-end. Meaning that it had not signature because we couldn't use the black box for it.
Now, the refunds are made from the front-end and thus we can send it to the black box and have a signature.)